README.rst

```diff
@@ -16,7 +16,55 @@ TODO
 
 Migrate existing projects
 --------------------------------
-TODO
+#. Use ruff.
+   #. Pull in the bare minimum ``pyproject.toml`` needed to use ruff.
+   #. Make your codebase pass with ruff.  Commit after each step:
+      #. ``ruff check --fix``
+      #. ``ruff check --fix --unsafe-fixes``
+      #. ``ruff check --add-noqa``
+      #. ``ruff format``
+   #. Replace use of black, flake8, pydocstyle, isort, and pylint in Makefile/CI
+      with:
+      - ``ruff check --fix``
+      - ``ruff format``
+#. Modify top-level files in your project to match what's in Starbase as closely
+   as possible.
+   #. ``Makefile`` - Ensure you use ``uv`` and at least have the same targets.
```

Comment:
We can probably call out specific targets:

- `setup` (and actually I'd love to distinguish `setup-lint` and `setup-test` so we can use those in CI and not have to wait for extra installs)
- `lint`
- `test-unit`
- `test-integration`
- `coverage`

Reply:
Good call, updating

Reply:
Also we're definitely thinking along the same lines for the setup target: `LINK REDACTED` see the repo I usually work in, issue #443.